Error: FAIL
    at f1 (/scripts/async-await-follow.js:r:c)
    at Function.resolve (<anonymous>)
    at f1 (/scripts/async-await-follow.js:r:c)
    at <anonymous>
    at f1 (/scripts/async-await-follow.js:r:c)
    at f2 (/scripts/async-await-follow.js:r:c)
    at Function.resolve (<anonymous>)
    at f2 (/scripts/async-await-follow.js:r:c)
    at <anonymous>
    at f2 (/scripts/async-await-follow.js:r:c)
    at f3 (/scripts/async-await-follow.js:r:c)
    at process._tickCallback (internal/process/next_tick.js:r:c)
    at Function.Module.runMain (internal/modules/cjs/loader.js:r:c)
    at Function.resolve (<anonymous>)
    at f3 (/scripts/async-await-follow.js:r:c)
    at <anonymous>
    at f3 (/scripts/async-await-follow.js:r:c)
    at Object.<anonymous> (/scripts/async-await-follow.js:r:c)
    at Module._compile (internal/modules/cjs/loader.js:r:c)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:r:c)
    at Module.load (internal/modules/cjs/loader.js:r:c)
    at tryModuleLoad (internal/modules/cjs/loader.js:r:c)
    at Function.Module._load (internal/modules/cjs/loader.js:r:c)
    at Function.Module.runMain (internal/modules/cjs/loader.js:r:c)
    at startup (internal/bootstrap/node.js:r:c)
    at bootstrapNodeJSCore (internal/bootstrap/node.js:r:c)
